packages/react-examples/src/react/Button/Button.IconWithTooltip.Example.tsx

```diff
@@ -35,8 +35,9 @@ export const ButtonIconWithTooltipExample: React.FunctionComponent<IButtonExampl
         id={tooltipId}
         calloutProps={calloutProps}
         styles={hostStyles}
+        setAriaDescribedBy={false}
       >
-        <IconButton iconProps={emojiIcon} title="Emoji" ariaLabel="Emoji" disabled={disabled} checked={checked} />
+        <IconButton iconProps={emojiIcon} aria-describedby={tooltipId} disabled={disabled} checked={checked} />
```

Comment:
Does a tooltip as aria-describedby actually work? I thought I remembered @smhigley mentioning that it didn't (but I might be mixing it up with something else).

Reply:
It works in my testing but I'd appreciate @smhigley 's input here.

You may be thinking about the automatic `aria-describedby` that `Tooltip` applies to a wrapper `div` which is incorrect as [noted in this comment](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/Tooltip/TooltipHost.base.tsx#L99).